### PR TITLE
[NO JIRA] Keep sql and drop test files

### PR DESCRIPTION
**Pull request by GarrettEHill**
## Description of change

- sql files are needed for the ssdi
- testing files should not be on deployed env

## Issue(s)

* https://jira.acf.gov/browse/TTAHUB-0

### Every PR

- [ ] Meets issue criteria
- [ ] JIRA ticket status updated
- [ ] Code is meaningfully tested
- [ ] Meets accessibility standards (WCAG 2.1 Levels A, AA)
- [ ] API Documentation updated
- [ ] Boundary diagram updated
- [ ] Logical Data Model updated
- [ ] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions
- [ ] UI review complete

### Before merge to main

- [ ] OHS demo complete
- [ ] Ready to create production PR

### Production Deploy

- [ ] Staging smoke test completed

### After merge/deploy

- [ ] Update JIRA ticket status

### CODE REVIEW

File: .cfignore [hunk 1]
```diff
@@ -12,3 +12,5 @@ hses.zip
 temp/
 tests/
 .tmp/
+*.test.js
+!*.sql
```
Comment (thewatermethod) [inline .cfignore:16]: does this do it? Doesn't it need to be in the deploy config?

Comment (GarrettEHill) [reply .cfignore:16]: according to chat gpt this should do it

Comment (thewatermethod) [reply .cfignore:16]: oh boy

Comment (thewatermethod) [reply .cfignore:16]: let's deploy to sandbox?

Comment (GarrettEHill) [reply .cfignore:16]: works: https://tta-smarthub-sandbox.app.cloud.gov/api/admin/ssdi/list-queries

**Comment by GarrettEHill** [comment]
./build before
```
      1 .css
      1 .html
      2 .jpg
    943 .js
      2 .json
      2 .map
      3 .png
      1 .svg
     23 .ttf
      2 .txt
     18 .woff
     19 .woff2
```

./build after
```
      1 .css
      1 .html
      2 .jpg
    746 .js
      2 .json
      2 .map
      3 .png
      1 .svg
     23 .ttf
      2 .txt
     18 .woff
     19 .woff2
```

**Review by thewatermethod — APPROVED** [review]

**Pull request merged**